lldbinit.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+# Initialization for lldb.
+
+__copyright__ = """
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""
+
+if __name__ == "__main__":
+    print("Run only as script from LLDB... Not as standalone program!")
```

Comment:
```suggestion
    exit("Run only as script from LLDB... Not as standalone program!")
```

Reply:
Switched to assert

Reply:
Our python checkers don't like me putting the assert above the imports. The `import lldb` will fail when run outside lldb anyway. So I just made it into a comment.